webapp/package.json

```diff
@@ -22,19 +22,28 @@
     "@sveltejs/adapter-auto": "^3.2.4",
     "@sveltejs/kit": "^2.5.27",
     "@sveltejs/vite-plugin-svelte": "^3.1.2",
+    "@tailwindcss/typography": "^0.5.14",
+    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/svelte": "^5.2.1",
+    "@testing-library/user-event": "^14.5.2",
     "@types/eslint": "^9.6.1",
     "@types/node": "^22.5.5",
     "@vitest/coverage-istanbul": "^2.1.1",
+    "autoprefixer": "^10.4.20",
+    "carbon-components-svelte": "^0.85.2",
     "eslint": "^9.10.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-svelte": "^2.44.0",
     "globals": "^15.9.0",
+    "happy-dom": "^15.7.4",
     "nyc": "^17.0.0",
     "playwright-test-coverage": "^1.2.12",
     "prettier": "^3.3.3",
     "prettier-plugin-svelte": "^3.2.6",
+    "prettier-plugin-tailwindcss": "^0.6.5",
     "svelte": "^4.2.19",
     "svelte-check": "^4.0.2",
+    "tailwindcss": "^3.4.9",
```

Comment:
We can remove the tailwind dependencies if we are not using it!

Reply:
yeah will do